README.md

```diff
@@ -6,12 +6,19 @@ Als organisatorischen Rahmen für das Projekt nutzen wir GitHub Classroom. Inhal
 
 ## How to use...
 
+No matter how you run the application, the `./data` folder must be created and the images-folders must be added before starting. 
```

Comment:
Der `./data` Ordner liegt mit im Repository und muss meines Wissens nach somit nicht noch erstellt werden.

Reply:
Ne hab den gelöscht und ignored und somit müsste der Weg sein, bei dir musste ich ihn auch erstellen, wenn dann müsste man glaub in gitignore schreiben `data/*` und da ein `.keep` file anlegen weil leere Ordner pusht er ja nicht, bzw. zeigt sie nicht an

Reply:
Interessanter weiße hat er bei mir den leeren Ordner mit geladen :D war darum etwas irritiert. Aber wenn es seine Richtigkeit hat passt das für mich 👍